src/zebra/subscriptions.clj

```diff
@@ -0,0 +1,43 @@
+(ns zebra.subscriptions
+  (:refer-clojure :exclude [list])
+  (:require
+    [zebra.utils :refer [transform-params]])
+  (:import
+    (com.stripe.model
+      Subscription
+      StripeCollection)
+    (com.stripe.net
+      RequestOptions)
+    (java.util
+      Map)))
+
+(defn subscription->map
+  [^Subscription subscription]
+  {:id                   (.getId subscription)
+   :customer             (.getCustomer subscription)
+   :cancelled-at         (.getCancelAt subscription)
```

Comment:
It might be better to prefer :cancel-at here as the cancel at date might be in the future?